main.go

```diff
@@ -190,6 +191,10 @@ func main() {
 			log.Warnf("Sending the edit to review failed. Please change \"Retry changes without sending to review\" input to true if you wish to send the changes with the changesNotSentForReview flag. Please note that in that case the review has to be manually initiated from Google Play Console UI")
 		}
 	}
+	if strings.Contains(errorString, internalServerError) {
+		log.Warnf("Google Play API responded with an unknown error")
+		log.Warnf("Suggestion: create a release manually in Google Play Console, it might reveal the real error")
```

Comment:
This is just a nitpicking comment.

The `it might reveal the real error` part felt a bit disconnected from the first part of the sentence. I cannot really describe why it felt that way. 😄 

I would propose
```
Suggestion: create a release manually in Google Play Console because the UI has the capability to present the underlying error in certain cases
```

Reply:
I like your better too, let's go with that